application/config.json.template

```diff
@@ -88,4 +88,9 @@
     ],
     "logInfoChannelWebhook": "<put_your_webhook_here>",
     "logErrorChannelWebhook": "<put_your_webhook_here>"
+    "starboard": {
+        "oofEmojiName": "oof"
+        "lmaoEmojiName": "lmao"
+        "starboardChannelId": <put_channel_id_here>
```

Comment:
name please, not ID. see other channels in the config (its easier to use and will for most work out of the box)